### DISCUSSION TITLE
Blogpost to tell the community about progress since our initial announcement

### DISCUSSION THREAD

**Discussion by dahavey** [microsoft/ebpf-for-windows#645] [Announcements]
Since our initial blogpost announcing the project we have had a lot of activity, but the community is largely unaware of this. Unless they are attending our triage meetings or have heard one of Dave's talks they probably think that nothing is going on. We need to produce a second blogpost detailing progress, future directions, etc.

Keep the community engaged and excited about eBPF on Windows!

**Comment by dahavey** [comment]
We should probably do this at least once or twice a year.

**Comment by dthaler** [comment]
Blog post is on track to appear on Monday November 29.

**Reply by dthaler** [reply]
Blog post is now live at https://cloudblogs.microsoft.com/opensource/2021/11/29/progress-on-making-ebpf-work-on-windows/
